src/odin/enhancedtrippath.cc

```diff
@@ -891,6 +929,109 @@ std::string EnhancedTripLeg_Edge::SignElementsToParameterString(
 
   return str;
 }
+
+std::string EnhancedTripLeg_Edge::TurnLanesToString(
```

Comment:
debug output to see turn lane info on each edge - it will like: `turn_lanes=left|left;through|through;right`